scripts/utils/index.ts

```diff
@@ -42,7 +42,7 @@ export enum DEPLOYMENT_SALTS {
   MULTI_SEND_CALLONLY = "DEVX_MULTI_SEND_CALLONLY_V0_11042023_pcnXVXc",
   WALLET_FACTORY = "DEVX_WALLET_FACTORY_V0_11042023_vyLkpGh",
   WALLET_IMP = "DEVX_WALLET_IMP_V0_11042023_AwPKF0R",
-  SINGELTON_PAYMASTER = "DEVX_SINGLETON_PAYMASTER_V0_11042023_v6ISI9i",
+  SINGELTON_PAYMASTER = "DEVX_SINGLETON_PAYMASTER_V1_03082023_0Af0vtw", // 0x0000064e9c653e373af18ef27f70be83df5476b7
```

Comment:
Shouldn't this be new field SINGLETON_PAYMASTER_V2?

What if we need to deploy v1 paymaster on other new chains? If not then on dashboard we have to consider showing only V2 as version for paymasters.

Reply:
It starts with 0 so I incremented it. on dashboard @divyaN73 suggested 1.1.0 which makes sense as it's not major update. besides you can change date or anything on the salt to deploy v1/any on other new chains

Reply:
also using this salt it's already deployed